### ISSUE TITLE
[0.4.0] [Progression] Cave Illuminator

### ISSUE THREAD

**Issue by Hjaldar** [xt9/cryogenic-simulation#15]
TL;DR It's OP and access to it should not be so easy.

**Story time:**
I got to the point in Lunar Plane progression where I needed a good supply of Sooty Marble. I'd done some exploring on the surface and had obtained a small number of Sooty Marble blocks from a dark shrine, but after a close shave with an Evoker I decided that was not a safe method of obtaining Sooty Marble. The lack of a recipe was going to send me underground for more (which I think is great).

I made a Luminous Crafting Table, a Resonating Wand and a Cave Illuminator from easily obtained surface resources. At the start of the next Minecraft day I plonked the Cave Illuminator down on the surface next to a nearby dungeon tower, waited a couple of minutes, and headed down. In my journey to the bottom, looting each level as I went, I met exactly one mob from a spawner (a creeper on level 4). I had to deal with some trolls as I went and place a few torches in some small still-dark areas. With a Tinkers steel sword, diamond armour and no bow there was effectively zero risk but full dungeon's worth of loot as a reward. Plus I got almost 1k Sooty Marble blocks from mining most of the floor on the bottom level. I think this cheapened the experience.

**Suggested possible fixes:**
1) Remove the recipe for the Cave Illuminator, potentially give it an Atlas Tip and add it to Lunar Plane dungeon loot in the reward chests, with a weight of 1, so it will have similar rarity to Advanced Backpacks, etc. This will mean the first couple of dungeons will need to be experienced as intended but eventually they will becoming much easier.
2) Change the recipe for the Cave Illuminator to move it to the end of the Lunar Plane progression. This could be done, for example, by replacing the Runed Marble in the recipe with Brittle Plastic. Probably means the player will need to do at least one dungeon or go caving in the Lunar Plane before they can get a Cave Illuminator.
3) Change the recipe for the Cave Illuminator to move it later in the progression to a point where it won't trivialise dungeons in later dimensions.
4) Disable the Cave Illuminator completely (not my preferred option, see Notes below).
5) ???

Notes:
- the Cave Illuminator needs sky access and daylight to function.
- it has a search radius of 64 blocks, so works in a 128x128x128 volume. This is hard-coded.
- lights are only placed in caves, i.e. locations with no sky access
- 4 Cave Illuminators can be used to eventually completely light up the caves to a distance of 128 blocks around the player, allowing very efficient mob farms in normal world gen.

**Comment by xt9** [comment]
Wow that's an interesting use of it, sounds like a needed change.

**Comment by xt9** [comment]
Fixed in 46773ff2dbea4b9aa2ec8b152c11dc4296727356
